cdap-ui/app/tracker/controllers/results-ctrl.js

```diff
@@ -137,6 +137,14 @@ class TrackerResultsController {
       scope: this.$scope
     };
 
+    if(params.query === '*') {
+      params.sort = 'creation-time desc';
+      params.numCursors = 10;
+    } else {
+      params.query = params.query + '*';
+      params.target = 'app';
```

Comment:
As provided on https://issues.cask.co/browse/TRACKER-177, it should be in params when we dont search with *
Could you please confirm